src/web3client/liquidator.js

```diff
@@ -181,6 +185,11 @@ class Liquidator {
 
   async sendWithRetry (wallet, txObject, ms) {
     await this.app.timer.timeout(1000);
+    // if txObject hit gas price limit we should timeout with additional time
+    if (txObject.hitGasPriceLimit) {
```

Comment:
this is over complicated, lets liquidation job continue to run without timeouts inside timeouts